src/shared/utils/camelizeKeys.ts

```diff
@@ -1,6 +1,8 @@
 import camelCase from 'lodash/camelCase'
 
-export function camelizeKeys(obj = {}) {
+export function camelizeKeys(
+  obj: Record<string, any> = {}
```

Comment:
Slightly stricter/more relevant typing than any, hoping this doesn't come back to bite but Ill keep an eye on it